packages/fiori/src/themes/IllustratedMessage.css

```diff
@@ -18,7 +18,7 @@
 }
 
 .ui5-illustrated-message-illustration {
-    margin-bottom: 2rem;
+    margin: 2rem 0;
```

Comment:
On size “spot” there is greater margin on the illustration coming from this rule.